arduino/main/lib/src/SerialCommunication.cpp

```diff
@@ -1,207 +1,197 @@
 #include "SerialCommunication.h"
 
 // Package structure definitions
-#define PACKAGE_HEADER		0xCA	// Jeff da CA!
-#define PACKAGE_EOL			0x0A	// \n
-#define PACKAGE_DATA_MAX	10
-
+#define PACKAGE_HEADER 0xCA // Jeff da CA!
+#define PACKAGE_EOL 0x0A    // \n
+#define PACKAGE_DATA_MAX 10
 
 // GET and SET commands
-#define GET_SPEED 			0
-#define GET_ANGLE 			1
-#define GET_SPEEDANGLE 		2
-#define GET_ULTRAREADER 	3
-
-#define SET_SPEED 			100
-#define SET_ANGLE 			101
-#define SET_SPEEDANGLE 		102
+#define GET_SPEED 0
+#define GET_ANGLE 1
+#define GET_SPEEDANGLE 2
+#define GET_ULTRAREADER 3
 
-#define GET_STATUS 			200
-#define ACK_ERROR			254
-#define ACK_OK				255
+#define SET_SPEED 100
+#define SET_ANGLE 101
+#define SET_SPEEDANGLE 102
 
+#define GET_STATUS 200
+#define ACK_ERROR 254
+#define ACK_OK 255
 
 // Error codes
-#define ERROR_PKG_SIZE		1		// Validation: Package too small
-#define ERROR_PKG_HEADER	2		// Validation: Invalid package header
-#define ERROR_PKG_CHKSUM	3		// Validation: Invalid checksum
-#define ERROR_UNKN_CMD		11		// Package content: Undefined command
-#define ERROR_DATA_SIZE		12		// Package content: Invalid data size for the current command
-#define ERROR_OUT_OF_BOUND	13		// PAckage content: Value too large
-
-
-int SerialCommunication::readBuffer(byte * buffer, int bufferSize){
-	return Serial.readBytesUntil(PACKAGE_EOL, buffer, bufferSize);
+#define ERROR_PKG_SIZE 1   // Validation: Package too small
+#define ERROR_PKG_HEADER 2 // Validation: Invalid package header
+#define ERROR_PKG_CHKSUM 3 // Validation: Invalid checksum
+#define ERROR_UNKN_CMD 11  // Package content: Undefined command
```

Comment:
Not sure if you like this better. I think it's easier right now to just stick to clang-format.